src/main/java/com/criticalsoftware/User.java

```diff
@@ -0,0 +1,55 @@
+package com.criticalsoftware;
+import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import io.quarkus.mongodb.panache.common.MongoEntity;
+import java.time.LocalDate;
+
+@MongoEntity(collection = "users")
+public class User extends PanacheMongoEntity {
+
+    private String name;
+    private LocalDate dateBirth;
+    private Contact contact;
+
+    public User() {
+
```

Comment:
Can we try without the empty constructor?

Reply:
Doesn't work without.